### PR TITLE
Fix: header covers part of elements when scrolling moves

### LINKED ISSUE #1: Improve anchors
When clicking on `About` or when trying to submit with a missing field, the scroll moves to a position in which the header covers part of the element:

image

image

We should avoid this behavior.

### PR DESCRIPTION
**Pull request by nicolas-justin**
Closes #1 

This pull request fixes the #1 issue.

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -57,3 +63,9 @@ a:hover {
     font-size: 1.2rem;
   }
 }
+
+@media screen and (prefers-reduced-motion: no-preference) {
+  html {
+    scroll-behavior: smooth;
+  }
+}
```
Comment (macabeus) [inline src/app.css:71]: Missing blank link at the EOF
```suggestion
}

```

**Comment by macabeus** [comment]
Thanks for the PR! Can you allow the `Allow edits and access to secrets by maintainers`, or apply my suggestions, please?

image

**Comment by nicolas-justin** [comment]
> Thanks for the PR! Can you allow the `Allow edits and access to secrets by maintainers`, or apply my suggestions, please?
> 
> image

This option is already active, but the label is "Allow edits by maintainers".
Anyway, I applied your suggestions.

**Comment by macabeus** [comment]
Thanks! I just merged and deployed your changes! 😄

**Pull request merged**